### PR TITLE
refactoring router for better seperation of concerns

### PR DESCRIPTION
**Pull request by dfirebaugh**
- refactoring router for better seperation of concerns
- improving the devcontainer so that it's easy to seed data with generators.  It will now auto-add a test user and some fake members if you run `make seed`
- removing seedLocal.sh
- adding some troubleshooting notes to the README
- fixing issue where subscriptionID wasn't being added in AddMember

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	"memberserver/config"
+	"memberserver/datastore"
+	"memberserver/resourcemanager"
+	"memberserver/resourcemanager/mqttserver"
+
+	"github.com/shaj13/go-guardian/v2/auth/strategies/jwt"
+	"github.com/shaj13/go-guardian/v2/auth/strategies/union"
+)
+
+// type Router interface {
```
Comment (pacna) [inline api/api.go:14]: Can we add a comment here to explain why this is commented? If there is no reason then would it be alright to remove it?

File: api/auth/auth.go [hunk 2]
```diff
@@ -0,0 +1,224 @@
+package auth
```
Comment (pacna) [inline api/auth/auth.go:1]: this is good :+1:

File: api/router/info.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import (
+	"net/http"
+)
+
+type InfoHTTPHandler interface {
+	GetInfo(http.ResponseWriter, *http.Request)
+}
+
+func (r Router) setupInfoRoutes(infoServer InfoHTTPHandler) {
+	// swagger:route GET /api/resource resource getResourceRequest
```
Comment (pacna) [inline api/router/info.go:12]: Would it be alright if we could remove this? This is for `info`. Not `resource`

File: api/router/router.go [hunk 4]
```diff
@@ -0,0 +1,72 @@
+// Package Classification Member Server API.
```
Comment (pacna) [inline api/router/router.go:1]: So much cleaner. Well done.

**Review by pacna — COMMENTED** [review]
A few small comments. Also, make sure to run `sh gendocs.sh` so that we'll have the latest update to swagger

File: api/api.go (hunk 1 above)
Comment (dfirebaugh) [reply api/api.go]: removed.

File: api/auth/auth.go (hunk 2 above)
Comment (dfirebaugh) [reply api/auth/auth.go:1]: thanks

File: api/router/router.go (hunk 4 above)
Comment (dfirebaugh) [reply api/router/router.go:1]: thanks

File: api/router/info.go (hunk 3 above)
Comment (dfirebaugh) [reply api/router/info.go]: removed.

I think `info` was something from when I was setting up the API originally.  It just returns a helloWorld.  So, I will remove it.

**Comment by dfirebaugh** [comment]
done.  Let me know if you find anything else.

**Review by pacna — APPROVED** [review]

**Pull request merged**